iHome/ihome.frontend/src/app/components/add-device/add-device-dialog.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 mat-dialog-title>Devices to configure</h1>
+<div mat-dialog-content>
+  <app-available-device *ngFor="let device of devicesToConfigure" [deviceToConfigure]=device [roomId]="roomId"></app-available-device>
+</div>
+<mat-dialog-actions align="end">
+  <button mat-button mat-dialog-close>Cancel</button>
```

Comment:
Wydaje mi się że jak uzywamy ikonek to już do obu buttonów a nie jeden ikonka a jeden napis, nie wiem jak to wygląda w przeglądarce ale wydaje mi się że 2 ikonki albo 2 napisy będą wyglądały lepiej

Reply:
Ty teraz widzę że tej ikonki to nie powinno tutaj być nawet